### ISSUE TITLE
Remove attributes

### ISSUE THREAD

**Issue by Toobian** [vdemedes/mongorito#45]
How can i do to remove attribute ?
i create temporary attributes like token to recover password. But after used this attribute i want to remove from my Document.
I see nothing in set method which remove attribute if nothing is passed or null value...

**Comment by vdemedes** [comment]
As a workaround, you can:

```javascript
delete post.attributes.title;
yield* post.save();
```

Thanks for suggestion, I will add `remove()` or `unset()` method.

**Comment by Toobian** [comment]
Thanks for solution :)

**Comment by Achrome** [comment]
The way I have dealt with this is the following. I'm thinking that this might also work on the core `Model`

```js
export default class User extends Model {
  configure() {
    this.privateFields = ['password', 'created_at', 'updated_at'];
  }
  filter() {
    return R.omit(this.privateFields, this.attributes); // R is an instance of ramda.js
  }
  toJSON() {
    return this.filter();
  }
}
```

Let me dig around the source and see if I can create a PR if this sounds good.

**Comment by vdemedes** [comment]
Added `unset()` method, usage:

```javascript
user.unset('email');
post.unset('author.name');
```

Available on npm as `mongorito@1.3.4`.
